stable/kommander/requirements.lock

```diff
@@ -4,13 +4,13 @@ dependencies:
   version: 0.1.10
 - name: kommander-federation
   repository: https://mesosphere.github.io/yakcl/charts
-  version: v0.1.0
+  version: 0.1.2
```

Comment:
This seems unintended. Need to find out why that "v" prefix got changed.

Reply:
nvm, this file is auto-generated, anyway.